packages/schematics/angular/migrations/update-8/differential-loading_spec.ts

```diff
@@ -74,5 +74,15 @@ describe('Migration to version 8', () => {
       expect(tree2.readContent('/browserslist'))
         .toContain('Googlebot support');
     });
+
+    it('should move browserslist file if it exists in the sourceRoot', () => {
+      tree.create('/src/browserslist', 'last 2 Chrome versions');
+      tree.delete('/browserslist');
+      const tree2 = schematicRunner.runSchematic('migration-07', {}, tree.branch());
```

Comment:
I am confused why these all say `migration-07`

Reply:
It's the migration name. For example for Angular CLI version 8, we have multiple migrations.

Reply:
https://github.com/alan-agius4/angular-cli/blob/42cfb32469dc51e651e6e07763878bf935a92cf3/packages/schematics/angular/migrations/migration-collection.json#L33-L41